bluebrain/repo-bluebrain/packages/spatial-index/package.py

```diff
@@ -41,11 +40,11 @@ class SpatialIndex(PythonPackage):
     depends_on("py-mvdtool~mpi", type=("build", "run"))
     depends_on("py-numpy-quaternion", type=("build", "run"), when="@0.2.1:")
     depends_on("py-numpy", type=("build", "run"))
-    depends_on("py-tqdm", when="@0.5.3:")
+    depends_on("py-tqdm", type=("build", "run"), when="@0.5.3:")
 
     depends_on("boost@1.79.0:", when="@0.5.2:")
```

Comment:
stupid remark, but it would seem nicer if the Boost stuff is right next to each other

Reply:
Yes, definitely.